helm/charts/hpe-csi-driver/README.md

```diff
@@ -52,7 +52,7 @@ helm repo update
 Install the latest chart:
 
 ```
-helm install hpe-csi hpe/hpe-csi-driver --namespace kube-system -f myvalues.yaml
+helm install hpe-csi hpe/hpe-csi-driver --namespace hpe-storage -f myvalues.yaml
```

Comment:
Could we add `kubectl create ns hpe-storage` before the install examples?
Could also try match the repo name and release name that is on actual [artifacthub](https://artifacthub.io/packages/helm/hpe-storage/hpe-csi-driver?modal=install)